pyaedt/edb_core/nets.py

```diff
@@ -1185,42 +1184,11 @@ def merge_nets_polygons(self, net_list):
             net name of list of net name.
```

Comment:
```suggestion
            Net name of list of net name.
```
I have no idea what this means. Should it be changed to something like "Name of net created from a list of nets"? instead.